torch_frame/datasets/huggingface_dataset.py

```diff
@@ -0,0 +1,139 @@
+from __future__ import annotations
+
+import pandas as pd
+
+import torch_frame
+from torch_frame import stype
+from torch_frame.utils.split import SPLIT_TO_NUM
+
+
+class HuggingFaceDatasetDict(torch_frame.data.Dataset):
+    r"""Load a Hugging Face :obj:`datasets.DatasetDict` dataset
+    to a :obj:`torch_frame.data.Dataset` with pre-defined split information.
+
+    Args:
+        path (str): Path or name of the dataset.
+        name (str, optional): Defining the name of the dataset configuration.
+
+    Example:
+        Load the `spotify-tracks-dataset` dataset from the Hugging Face Hub
+        to the :obj:`torch_frame.data.Dataset`:
```

Comment:
Can we give a link to the HuggingFace  dataset dict so that users know what `path` to use?